client/src/Components/AddVideo.js

```diff
@@ -0,0 +1,115 @@
+import { useState } from "react";
+const AddVideo = ({ setRebuild, setVideoCards }) => {
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (
+      title.trim() &&
+      url.trim() &&
+      url.startsWith("https://www.youtube.com/")
+    ) {
+      const newVideo = {};
+      newVideo.title = title;
+      newVideo.url = url;
+      console.log(newVideo);
+      fetch("https://video-recomendations-server.onrender.com/videos", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(newVideo),
+      })
+        .then((res) => res.json())
+        .then((data) => {
+          setVideoCards(data);
+          setRebuild((rebuild) => rebuild + 1);
+        })
+        .catch((error) => console.log(error));
+    } else {
+      alert("Please fill fields correctly");
```

Comment:
It can be more useful friendly to tell them which fields were incorrect, and what was expected, e.g. if I'd put a non-youtube link as the URL, telling me that you expect a youtube link would be helpful :)